scripts/hooks/getChatLogEntryContext.js

```diff
@@ -1,8 +1,81 @@
 /**
  * Add right click option to damage chat cards to allow application of damage
+ * Add right click option to use fortune point on own rolls
  */
 Hooks.on("getChatLogEntryContext", (html, options) => {
   let canApply = li => li.find(".opposed-card").length;
+  let canApplyFortuneReroll = function(li){
+    //Condition to have the fortune contextual options:
+    //Have a selected character
+    //Have fortune point
+    //Own the roll
+    //Once per roll (or at least, not on a reroll card)
+    //Test must be failed 
+    //If opposed, test must be over
```

Comment:
Is this really necessary?

1. It's confusing, players might be wondering why they can't right click reroll
2. Where exactly is this rule in the book? All I see is "Failed Test"
![image](https://user-images.githubusercontent.com/28637157/80268282-77ef1000-866b-11ea-85a6-9a4c32991e0f.png)

I just can't imagine anything lost by allowing them to reroll before the opposed test is over.

Reply:
You are not allowed to reroll a test if it is not failed. An opposed test is two rolls. You can't reroll a "bad" roll if you win the opposed test